Go 1.15: panic: cannot create context from nil parent

Go 1.15 rc 1 on Fedora Rawhide:

```
github.com/pengsrc/go-shared/log
[2020-08-01T03:12:07.933Z #4120132] DEBUG -- : Singleton logger 
[2020-08-01T03:12:07.933Z #4120132] DEBUG -- : Debug message test, hi Apple 
--- FAIL: TestNewLogger (0.00s)
panic: cannot create context from nil parent [recovered]
	panic: cannot create context from nil parent
goroutine 55 [running]:
testing.tRunner.func1.1(0x55da7e8a9c40, 0x55da7e8ed590)
	/usr/lib/golang/src/testing/testing.go:1057 +0x30d
testing.tRunner.func1(0xc000328780)
	/usr/lib/golang/src/testing/testing.go:1060 +0x41a
panic(0x55da7e8a9c40, 0x55da7e8ed590)
	/usr/lib/golang/src/runtime/panic.go:969 +0x175
context.WithValue(0x0, 0x0, 0x55da7e8a92c0, 0x55da7e8ed9c0, 0x55da7e8a9c40, 0x55da7e8ed9d0, 0x0, 0x0)
	/usr/lib/golang/src/context/context.go:521 +0x188
github.com/pengsrc/go-shared/log.TestNewLogger(0xc000328780)
	/builddir/build/BUILD/go-shared-1999055a4a14e23c8c441fbbfbbd2635998fd71a/_build/src/github.com/pengsrc/go-shared/log/logger_test.go:44 +0x365
testing.tRunner(0xc000328780, 0x55da7e8ecb68)
	/usr/lib/golang/src/testing/testing.go:1108 +0xef
created by testing.(*T).Run
	/usr/lib/golang/src/testing/testing.go:1159 +0x38d
exit status 2
FAIL	github.com/pengsrc/go-shared/log	0.011s
```

--- a/log/logger_test.go
+++ b/log/logger_test.go
@@ -41,7 +41,7 @@ func TestNewLogger(t *testing.T) {
 	// Test logging for context.
 	type contextKey string
 	const traceID contextKey = "trace_id"
-	ctx := context.WithValue(nil, traceID, "60b725f10c9c85c70d97880dfe8191b3")
+	ctx := context.WithValue(context.Background(), traceID, "60b725f10c9c85c70d97880dfe8191b3")
 
 	l.SetInterestContextKeys([]interface{}{traceID})
 

